helm/ako/crds/ako.vmware.com_aviinfrasettings.yaml

```diff
@@ -52,6 +52,10 @@ spec:
                     type: boolean
                   enablePublicIP:
                     type: boolean
+                  enableHTTP2:
```

Comment:
@arihantg, @akshayhavile,
Do we have to make this port specific? In the controller, http2 can be enabled per port.